packages/stash-typedoc/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "A package for creating API docs, to publish at https://docs.cipherstash.com/tsdoc",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\""
```

Comment:
Removing `exit 1` here so that running `pnpm test` for stash-typedoc doesn't cause CI to fail.